src/utils/stringFormat.tsx

```diff
@@ -48,3 +50,63 @@ export function shortHand(
 ): string {
   return shortHandNames[name] || name;
 }
+
+/**
+ * Translates true/false to Yes/No strings
+ * @param str the string to translate
+ */
+export function booleanToString(str: string): string {
+  switch (str) {
+    case 'true':
+      return 'Yes';
+    case 'false':
+      return 'No';
+    default:
+      return str;
+  }
+}
+
+/**
+ * Compares 2 main entries and returns the relative position of the first to the second.
+ * @param e1 the first entry
+ * @param e2 the second entry
+ */
+export function compareMainEntries(e1: Entry, e2: Entry): number {
+  if (MainSiteEntryOrder[e1.title] < MainSiteEntryOrder[e2.title]) {
+    return -1;
+  } else {
+    return 1;
+  }
+}
+
+/**
+ * If both are present in the given list of entries, combines genus and species into a scientific name.
+ * @param entries the list of entries
+ */
+export function combineScientificName(entries: Entry[]): Entry[] {
+  const newEntries: Entry[] = [];
+  let species;
+  let genus;
+  entries.forEach((entry: Entry) => {
```

Comment:
💭 There are a few places where we seem to be wrestling between arrays and records for site entries, it might be easier if transform the list into a record or map in the selector so that way we can do everything by name after

Reply:
Will look into this in a separate ticket